Kickstarter-iOS/Views/Controllers/PledgeContinueViewController.swift

```diff
@@ -76,10 +76,7 @@ final class PledgeContinueViewController: UIViewController {
 
 private let continueButtonStyle: ButtonStyle = { button in
   button
-    |> greenButtonStyle
-    |> UIButton.lens.title(for: .normal) %~ { _ in
-      Strings.Continue()
-    }
-    |> (UIButton.lens.titleLabel .. UILabel.lens.font) .~ UIFont.boldSystemFont(ofSize: 16)
-    |> (UIButton.lens.titleLabel .. UILabel.lens.lineBreakMode) .~ NSLineBreakMode.byTruncatingMiddle
+    |> ButtonStyleType.green.style
```

Comment:
Not necessary to use the `ButtonStyleType` `enum` here, that is really just for our VMs to output a button style as something that we can do equality comparison on in tests. Swift doesn't support equality comparison on function types. So this can just be `greenButtonStyle` 👍